sql/tiger_import_start.sql

```diff
@@ -32,7 +29,7 @@ BEGIN
 
   numberrange := endnumber - startnumber;
   rangestartnumber := startnumber;
-
+  
   IF (interpolationtype = 'odd' AND startnumber%2 = 0) OR (interpolationtype = 'even' AND startnumber%2 = 1) THEN
     startnumber := startnumber + 1;
     stepsize := 2;
```

Comment:
stepsize is used to filter out broken tiger data later, but rangestartnumber is unused.